src/clear.js

```diff
@@ -8,12 +8,12 @@ export const clearComplete = () => {
 
   wrapperChecks.forEach((item) => {
     item.remove();
+  });
 
-    const get = JSON.parse(localStorage.getItem('data'));
+  const get = JSON.parse(localStorage.getItem('data'));
```

Comment:
The local storage is working fine, however, there's an issue. When I clear all the completed tasks and refresh the page it renders an `undefined` task. Please refer to the attached image.

- [ ] Kindly fix this bug. Consider this: when the page loads, check if there's something on local storage using a conditional.

![image](https://user-images.githubusercontent.com/92215878/220995232-e2308aea-91f2-4f3b-adaa-823e242a4fa1.png)

Reply:
Okay, thank you for the correction.